docs/manuals/core/config/job-config.md

```diff
@@ -114,4 +114,71 @@ For example, the following configuration will allow Taipy to run up to eight `Jo
     max_nb_of_workers = "8:int"
     ```
 
+# Cluster mode
+
+!!! warning "Available in Taipy Enterprise edition"
+
+    This section is relevant only to the Enterprise edition of Taipy.
+
+The *cluster* mode is designed to make a Taipy application executes jobs on a remote cluster of
+distributed workers. The number of machines available in the cluster is unlimited, so the
+application becomes fully scalable horizontally. It can also be adjusted dynamically depending on
+some load metric.
+
+With the *cluster* mode, a `Job^` asynchronously runs in its own environment. At the submission,
+the job is queued in a RabbitMQ queue. When there is a worker available, the job is dequeued and
```

Comment:
Do we need to mention the technology (RabbitMQ) at this point?

Reply:
We probably not. What do you think @jrobinAV?

Reply:
I would say yes. 
We rely on RabbitMQ. Users need to know it to start it and manage it.